composer.json

```diff
@@ -48,7 +48,6 @@
         "components/font-awesome": "4.4.0",
         "ezsystems/comments-bundle": "~6.0.0",
         "ezsystems/content-on-the-fly-prototype": "~0.1.0",
```

Comment:
You need to remove the comma at the end of the previous line.

Hint: run `composer validate` before committing `composer.json` changes.